extensions/src/AWSSDK.Extensions.Configuration.AWSSystemsManager/AWSSDK.Extensions.Configuration.AWSSystemsManager.csproj

```diff
@@ -0,0 +1,12 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netstandard1.3</TargetFramework>
```

Comment:
Let's just start at netstandard2.0. Before 2.0 there are weird dependency references issue it would be nice to avoid and most people are just using netstandard2.0 anyway. It also reduces are test coverage requirements and I can't imagine there are very many people still using ASP.NET Core 1.x and would also be doing major work like changing their configuration

Reply:
funny, I actually had everything as 2.0 to start, but thought "I bet they will want all this to be backwards compatible as much as possible" :)  
I'll get this updated and adjust any packages or other items as needed

Reply:
Done

Reply:
The 1.x era is a pain so we want to push as much as possible to 2.0.